react_front_end/src/pages/components/GameName.js

```diff
@@ -29,7 +29,7 @@ export const GameName = () => {
       className={classes.title}
     >
       <p data-testid="Game Name">
-        <span style={{ fontWeight: "bold" }}>Game</span>: {`${gameInfo.game}`}
+        <span style={{ fontWeight: "bold" }}>Game</span>: {`${gameInfo.name}`}
```

Comment:
Change this back to ```{`${gameInfo.game}`}``` for the time being please.

Reply:
Can you explain why? gamInfo.game does not return anything in my testing.